orders/views.py

```diff
@@ -50,5 +52,10 @@ class DriverShipmentHistoryView(generics.GenericAPIView):
     def get(self, request, *args, **kwargs):
         user = request.user
         queryset = Orders.objects.filter(order_state="confirmed", driver_id=user.id)
```

Comment:
move the authentication logic from the serializers here so that the response will include the status code matching the returned value